varcode/effect_ordering.py

```diff
@@ -52,10 +52,18 @@ def top_priority_transcript_effect(effects):
         elif priority == best_priority:
             best_effects.append(effect)
 
-    def effect_cds_len(effect):
-        """
-        Returns length of coding sequence of transcript associated with effect
-        """
-        return len(effect.transcript.coding_sequence)
-
-    return max(best_effects, key=effect_cds_len)
\ No newline at end of file
+    if any(effect.transcript.complete for effect in best_effects):
```

Comment:
Looks like my comment had two parts!

Part of my thinking was the elegance of a single sort key.

But also, in terms of the difference in results: the results for the `if` clause can either contain both complete and incomplete transcript effects, or just complete transcript effects. Just curious about leaving the incomplete stuff in, but lower in the sort order.

Probably not worthy of too much conversationing :)